<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Название сорта</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>210</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Форма</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>170</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Степень обжарки</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="id">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>60</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="name">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>110</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="roasting">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>160</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>210</y>
     <width>251</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Молотый</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>В зёрнах</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>400</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить/изменить</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>260</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string> Описание вкуса</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="description">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>250</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>310</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Цена</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="price">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>300</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>360</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Объем упаковки</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="volume">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>350</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
